tests/test_cff.py

```diff
@@ -8,3 +10,76 @@ def test_parse_cff():
         cff_content = f.read()
     authors = get_cff_authors(cff_content)
     assert authors is not None
+
+
+def test_broken_cff():
+    cff_file_emptyish = b"""
+    cff-version: 1.2.0
+    message: "This is a CFF devoid authors or DOI"
+    """
+    cff_file_bad_syntax = b"""
+    cff-version: 1.2.0
+    title: gimie :
+    authors:
+      family-names: Doe
+        given-names: John
+        - family-names: Smith
+    given-names:
+    Jane
+        orcid: 0000-0001-2345-6789
+    """
+    cff_file_broken_orcid = b"""
+    cff-version: 1.2.0
+    title: gimie
+    authors:
+      - family-names: Doe
+        given-names: John
+        orcid: 0000-0001-2345-6789
+      - family-names: Smith
+        given-names: Jane
+        orcid: http://www.orcid.org/0000-0001-2345-6789
+    """
+    cff_file_author_without_orcid = b"""
+    cff-version: 1.2.0
+    title: gimie
+    authors:
+      - family-names: Doe
+        given-names: John
+    """
+
+    # empty graph = \n according to rdflib
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_emptyish)
+        .serialize(format="ttl")
+        == "\n"
+    )
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_bad_syntax)
+        .serialize(format="ttl")
+        == "\n"
+    )
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_broken_orcid)
+        .serialize(format="ttl")
+        == "\n"
+    )
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_author_without_orcid)
+        .serialize(format="ttl")
+        == "\n"
+    )
```

Comment:
```suggestion
    assert len(
        CffParser(subject=URIRef("https://example.org/"))
        .parse(data=cff_file_emptyish)
        ) == 0
    
    assert len(
        CffParser(subject=URIRef("https://example.org/"))
        .parse(data=cff_file_bad_syntax)
    ) == 0
    
    assert len(
        CffParser(subject=URIRef("https://example.org/"))
        .parse(data=cff_file_broken_orcid)
    ) == 0
    
    assert len(
        CffParser(subject=URIRef("https://example.org/"))
        .parse(data=cff_file_author_without_orcid)
    ) == 0
```
**comment:** relying on serialization conventions is kinda brittle (they can change the newline thing).
**suggestion:** actually test expectation on the graph structure itself

Reply:
**comment(non-blocking):** as the test logic is constant, if you want to make this even neater, you could use pytest fixtures instead to make a parametrized test. See e.g. https://github.com/sdsc-ordes/gimie/blob/1866420f39b15312e6f73cab60c73f8e3ca677d3/tests/test_github.py#L16